xml/depl_crowbar_extra-features.xml

```diff
@@ -0,0 +1,82 @@
+<?xml version="1.0" encoding="utf-8"?>
+<!DOCTYPE chapter
+[
+<!ENTITY % entities SYSTEM "entity-decl.ent">
+%entities;
+]>
+<chapter xmlns="http://docbook.org/ns/docbook" xmlns:xi="http://www.w3.org/2001/XInclude" xmlns:xlink="http://www.w3.org/1999/xlink" version="5.0" xml:id="sec.depl.adm_crowbar_extra-features">
+ <title>Customizing &crow;</title>
+  <para>
+   In large deployments with many nodes, there are always some nodes that are
+   in a fail or unknown state. New &barcl;s cannot be applied to them and
+   values cannot be updated in some &barcl;s that are already deployed. This
+   happens because &crow; will refuse to apply a &barcl; to a list of nodes if
+   they are not all in <literal>ready</literal> state.
+  </para>
+  <para>
+   To avoid having to manually take out nodes that are not
+   <literal>ready</literal>, there is a feature called <literal>skip
+   unready nodes</literal>. Instead of refusing to apply the &barcl;, it will
+   skip the nodes that it finds in any other state than
+   <literal>ready</literal>.
+  </para>
+  <para>
+   <emphasis role="bold">Enabling the Feature</emphasis>
+  </para>
+  <para>
+   In <filename>/opt/dell/crowbar_framework/config/crowbar.yml</filename> set
+   the option <literal>skip_unready_nodes</literal> to <literal>true</literal>.
+  </para>
+  <screen>default: &amp;default
+skip_unready_nodes:
+  enabled: false &lt;&lt;&lt; change to true</screen>
+  <para>
+   <emphasis role="bold">Roles Affected</emphasis>
+  </para>
+  <para>
+   All &Barcl; roles are affected. The default config file includes all the
+   roles that have been tested and found to be working. Adding roles to the
+   default list is not supported for the <literal>skip_unready_nodes</literal>
+   feature. Removing default roles is supported.
+  </para>
+  <para>
+   The list of currently supported roles to skip:
+  </para>
+  <screen>- bmc-nat-client
+- ceilometer-agent
+- deployer-client
+- dns-client
+- ipmi
+- logging-client
+- nova-compute-ironic
+- nova-compute-kvm
+- nova-compute-qemu
+- nova-compute-vmware
+- nova-compute-xen
+- nova-compute-zvm
+- ntp-client
+- provisioner-base
+- suse-manager-client
+- swift-storage
+- updater</screen>
+  <para>
+   <emphasis role="bold">Determining Which Nodes Were Skipped</emphasis>
+  </para>
+  <para>
+   Skipped nodes are logged to the &crow; log
+   (<filename>/var/log/crowbar/production.log</filename>) where you can search
+   for the text <literal>skipped until next chef run</literal>. This
+   will print the log lines where nodes were skipped, the name of the node, and
+   the &barcl; which was being applied.
+  </para>
+  <screen>&prompt.user;grep "skipped until next chef run" /var/log/crowbar/production.log</screen>
+  <para>
```

Comment:
I would remove this part and line 76 as there is not much info to obtain from that log, so it should not be communicated to the end user, its more for internal debugging.

Reply:
Are you suggesting to remove all of "Determining Which Nodes Were Skipped"?
Both production.log greps?

It seems like this is information that a user might want to know. I've taken out 'grep "skip_unready_nodes"', and left 'grep "skipped until next chef run"'